UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -316,21 +316,33 @@ public GMImage Image
         /// <summary>
         /// The width of the texture.
         /// </summary>
-        public int Width => _image.Width;
+        /// <remarks>
+        /// In the case of an invalid or missing image, this will always be <c>-1</c>.
+        /// </remarks>
+        public int Width => _image?.Width ?? -1;
 
         /// <summary>
         /// The height of the texture.
         /// </summary>
-        public int Height => _image.Height;
+        /// <remarks>
+        /// In the case of an invalid or missing image, this will always be <c>-1</c>.
+        /// </remarks>
+        public int Height => _image?.Height ?? -1;
 
         /// <summary>
         /// Whether this texture uses the QOI format.
         /// </summary>
+        /// <remarks>
+        /// In the case of an invalid or missing image, this will always be <c>false</c>.
```

Comment:
use `see langword` please instead of `c` for the false

Reply:
Fixed